modules/Material/Storage.qml

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
There has to be another way to do this without adding a new component, even if you need to create new properties in the existing components, creating global variables is not the solution.